tests/sklearn/test_sklearn_models.py

```diff
@@ -438,11 +443,6 @@ def check_offset(model_class, n_bits, x, y):
     """Check offset."""
     model = instantiate_model_generic(model_class, n_bits=n_bits)
 
-    # Offsets are not supported by XGBoost
-    if is_model_class_in_a_list(model_class, get_sklearn_tree_models(str_in_class_name="XGB")):
```

Comment:
all these removals (2 or 3 more below) are moved to where we initialize the tests (in mark.parametrize)